TheBTeam.ConsoleApp/MainMenu.cs

```diff
@@ -81,8 +85,17 @@ public static void ShowMainMenu()
                 }
                 else if (mainMenuItem[currentItem] == ("View users"))
                 {
-                    UserViewer.ViewUsers(tmpListUsers.UsersList);
-                    Console.ReadKey();
+                    if (tmpListUsers.UsersList.Count > 0)
+                    {
+                        UserViewer.ViewUsers(tmpListUsers.UsersList);
+                        Console.ReadKey();
+                    }
+                    else
+                    {
+                        Console.WriteLine($"No data users, Please add new user");
```

Comment:
No data, please add new user or
no users pleas add new user

Reply:
maby add sth like
--------------------------------------------------------------------------------------
and later too